### DISCUSSION TITLE
Help for multi-sig wallet in Solidity

### DISCUSSION THREAD

**Discussion by Leopard-CC** [community/community#33760] [Projects and Issues] [answered]
I'm currently developing one smart contract using Solidity
And I'm in need of multi signature for the security
I already searched but didn't get exact method :)

**Comment by cryptofan430** [comment, marked as answer]
Hi mate
You can simply use [https://gnosis-safe.io](service)
Or can implement smart contract from their github [https://github.com/gnosis/MultiSigWallet](github)
Cheers!

**Reply by Leopard-CC** [reply]
I already heard about that gnosis wallet
Currently I'm in need of Solidity cord not just a service

**Reply by cryptofan430** [reply]
You can use [https://github.com/community/community/discussions/github](url)

**Comment by ossanna16** [comment]
Great question @tikhan1019, I'm glad you found our community 🙂

**Reply by cryptofan430** [reply]
Very glad to connect with you @ossanna16 🙌
Hope to keep in touch especially for projects to work together 😊

**Reply by ossanna16** [reply]
Likewise 🚀
